daprdocs/content/en/operations/components/pluggable-components/pluggable-components-overview.md

```diff
@@ -0,0 +1,111 @@
+---
+type: docs
+title: "Overview"
```

Comment:
Do we need to repeat the name "Pluggable components" even being under the section named "Pluggable components"?

I did the same as the Pub/Sub section did

Reply:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/5839364/194703650-26a4746e-a8f2-431e-bc8f-b1bb65d0383a.png">

how it will look like

Reply:
Let's change the metadata to be:

>`title: "Pluggable components overview"`
>`linkTitle: "Overview"`